snippets/csharp/keywords/ReadonlyKeywordExamples.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
```

Comment:
nit: this and the System.Text namespace aren't actually used in the example, so I'd remove them.